usr/share/rear/layout/save/default/950_verify_disklayout_file.sh

```diff
@@ -10,6 +10,15 @@
 #   Error: Partition doesn't exist
 # cf. https://github.com/rear/rear/issues/1681
 #
+# THIS LIMITATION APPLIES TO parted NOT BEING ABLE TO SPECIFY PARTITIONS IN
+# BYTES ONLY.
+#
+
+# Test by using the parted version numbers...
+parted_version=$( get_version parted -v )
+
+test "$parted_version" || BugError "Function get_version could not detect parted version"
+version_newer "$parted_version" 2.0 && return 0
```

Comment:
With that `version_newer "$parted_version" 2.0 && return 0`
the whole disklayout.conf verification is skipped in all current cases
(even on my old SLES11 I have parted version 2.3)
but 950_verify_disklayout_file.sh
does not only check for non-consecutive partitions,
it also does basic syntax checks in its first part
and that basic syntax checks should be kept in any case.
Only the checks for non-consecutive partitions might be skipped for newer parted.
But actually no check needs to be skipped at all because the actual checks
do not show user messages or error out.
All what needs to be done is at the end of 950_verify_disklayout_file.sh
not not show messages about non-consecutive partitions and
to not error out in this case for newer parted versions.

Reply:
Indeed, just skipped all this without actually checking the code :-(